gordo_components/builder/build_model.py

```diff
@@ -308,6 +373,9 @@ def provide_saved_model(
     replace_cache: bool
         Forces a rebuild of the model, and replaces the entry in the cache with the new
         model.
+    cv_mode: str
```

Comment:
Should have the 3 possible values. You can either copy-paste the doc from build_model, or write something like pandas/numpy does: 
```
cv_mode: {'full_build', 'cross_val_only', 'build_only'}
    Do a full model build and calculation of cross-validation (cv) scores, only calculate  cv scores, or only do a model build without cv.
```